src/js/controllers/wizard.js

```diff
@@ -298,6 +298,10 @@ module.exports = function(app) {
         );
       };
 
+      $scope.isOnline = function () {
+        return window.navigator.onLine;
```

Comment:
that was exactly my plan, BUT: there's CORS and Access-Control-Allow-Origin, so I wrote a little jsonp-wrapper, as I didn't get it to work with other servers:

```
<?php
header('Content-Type: application/json');

function is_valid_callback($subject)
{
    $identifier_syntax
      = '/^[$_\p{L}][$_\p{L}\p{Mn}\p{Mc}\p{Nd}\p{Pc}\x{200C}\x{200D}.]*+$/u';

    $reserved_words = array('break', 'do', 'instanceof', 'typeof', 'case',
      'else', 'new', 'var', 'catch', 'finally', 'return', 'void', 'continue',
      'for', 'switch', 'while', 'debugger', 'function', 'this', 'with',
      'default', 'if', 'throw', 'delete', 'in', 'try', 'class', 'enum',
      'extends', 'super', 'const', 'export', 'import', 'implements', 'let',
      'private', 'public', 'yield', 'interface', 'package', 'protected',
      'static', 'null', 'true', 'false');

    return preg_match($identifier_syntax, $subject)
        && ! in_array(mb_strtolower($subject, 'UTF-8'), $reserved_words);
}


$json = json_encode(array('health'=>'UP'));

# JSON if no callback
if( ! isset($_GET['callback']) )
    exit($json);

# JSONP if valid callback
if(is_valid_callback($_GET['callback']))
    exit("{$_GET['callback']}($json)");

# Otherwise, bad request
header('status: 400 Bad Request', true, 400);
?>

```

where could we put some global configuration settings? Like that URL to ```https://weimarnetz.de/health?callback=JSON_CALLBACK```